src/shared/utilities/childProcess.ts

```diff
@@ -64,13 +64,12 @@ export interface ChildProcessResult {
  * - call and await run to get the results (pass or fail)
  */
 export class ChildProcess {
+    private static runningProcesses: Record<string, ChildProcess> = {}
```

Comment:
Nit: use a different naming convention for the static var? Probably `CONSTANT_CASE`?

Reply:
Usually I only use all caps for things with a static size and value that is known at 'build' time, which this isn't.
Though I've seen things like `INSTANCE` being used.